packages/rulesets/generated/spectral/az-common.js

```diff
@@ -185,33 +185,6 @@ function getProperties(schema) {
     }
     return properties;
 }
-function isSchemaEqual(a, b) {
```

Comment:
FYI `rush build` removed this, but `function isSchemaEqual` appears to be present in `az-arm.js` and `az-dataplane.js`.